pkgs/top-level/all-packages.nix

```
@@ -20967,6 +20967,7 @@ in
   mpvScripts = recurseIntoAttrs {
     convert = callPackage ../applications/video/mpv/scripts/convert.nix {};
     mpris = callPackage ../applications/video/mpv/scripts/mpris.nix {};
+    simple-mpv-webui = callPackage ../applications/video/mpv/scripts/simple-mpv-webui.nix {};
```

Perhaps:

```suggestion
    simple-webui = callPackage ../applications/video/mpv/scripts/simple-mpv-webui.nix {};
```

Or even:

```suggestion
    webui = callPackage ../applications/video/mpv/scripts/simple-mpv-webui.nix {};
```

It's up to your decision. The last option seems more concise :thinking: ... The reasoning is that `mpv` is already in the attribute path of this derivation.

---

I personally prefer to use the same name as in the GitHub repo in order to make searching easier, even if it implies repeating "mpv". I think this approach is used in other places, such as Python packages (e.g. `pythonPackages.python-dateutil`).